scale/job/test/configuration/interface/test_job_interface.py

```diff
@@ -788,6 +788,82 @@ def test_output_dir_in_command(self):
         job_command_arguments = job_interface.fully_populate_command_argument(job_data, job_environment, job_exe_id)
         self.assertEqual(job_command_arguments, job_output_dir, 'expected a different command from pre_steps')
 
+    def test_absent_required_file_in_command(self):
+        job_interface_dict, job_data_dict, job_environment_dict = self._get_simple_interface_data_env()
+
+        job_interface_dict['input_data'] = [{
+            'name': 'bad_name',
```

Comment:
For each of the unit tests, change "bad_name" to "input_file" (or "input_files" for files).
Then add `job_interface_dict['command_arguments'] = '${input_file}'` for each test right after `job_interface_dict, job_data_dict, job_environment_dict = self._get_simple_interface_data_env()`. Right now the command_arguments start as '', so the parameter replacement is not being tested.